Hunting Queries/SecurityAlert/AlertsOnHost.yaml

```diff
@@ -22,10 +22,10 @@ query: |
   let v_EndTime = suspiciousEventTime+6h;
   SecurityAlert
   | where TimeGenerated between (v_StartTime .. v_EndTime)
-  | where Computer contains v_Host
   // expand JSON properties
   | extend Extprop = parse_json(ExtendedProperties)
   | extend Computer = iff(isnotempty(toupper(tostring(Extprop["Compromised Host"]))), toupper(tostring(Extprop["Compromised Host"])), tostring(parse_json(Entities)[0].HostName))
+  | where Computer contains v_Host
   | extend Account = iff(isnotempty(tolower(tostring(Extprop["User Name"]))), tolower(tostring(Extprop["User Name"])), tolower(tostring(Extprop["user name"])))
   | extend IpAddress = tostring(parse_json(ExtendedProperties).["Client Address"]) 
   | project TimeGenerated, AlertName, Computer, Account, IpAddress, ExtendedProperties
```

Comment:
@v-vdixit please validate this hunting query template. seems missing few properties